go/test/endtoend/cluster/mysqlctld_process.go

```diff
@@ -111,6 +113,11 @@ func (mysqlctld *MysqlctldProcess) Start() error {
 	go func(mysqlctld *MysqlctldProcess) {
 		err := mysqlctld.process.Wait()
 		if !mysqlctld.exitSignalReceived {
+			errBytes, ferr := os.ReadFile(mysqlctld.ErrorLog)
+			if ferr != nil {
+				log.Errorf("Failed to read the mysqlctld error log file %q: %v", mysqlctld.ErrorLog, ferr)
+			}
```

Comment:
I feel like we should not continue with the line `120` if `ferr != nil`

Reply:
Yeah, I'll change that. Thanks!